core/api/v1/admin.proto

```diff
@@ -138,4 +150,8 @@ service KeyTransparencyAdmin {
       delete: "/v1/domains/{domain_id}:undelete"
     };
   }
+
+  // Fully delete soft-deleted domains that have been deleted before the specified timestamp.
```

Comment:
nit: Fully delete the domains that have been soft-deleted before the specified timestamp.